packages/components/pie-button/src/button.scss

```diff
@@ -137,15 +149,15 @@ $breakpoint-wide: 768px;
         @include p.button-interactive-states('--dt-color-black', 'transparent');
     }
 
-    &[variant='inverse'] {
+    &.o-btn--inverse {
```

Comment:
It could be useful for part of this change be to document our CSS naming conventions in the wiki. Would you be happy to do that? It could be a port from Fozzie, but I don't think we currently document for web components.

Reply:
Yeh, sure. I'll spin this up as a separate PR 👍

Reply:
Here you go https://github.com/justeattakeaway/pie/wiki/PIE-%E2%80%93-CSS-Naming-Scheme